CONTRIBUTING.md

```diff
@@ -1,60 +1,9 @@
-# Contribution guide
+# Contribution to this repository
 
-![github-profile](https://user-images.githubusercontent.com/10350960/166113119-629295f6-c282-42c9-9379-af2de5ad4338.png)
+Thanks for your interest in contributing to MDN!
 
-- [Ways to contribute](#ways-to-contribute)
-- [Finding an issue](#finding-an-issue)
-- [Asking for help](#asking-for-help)
-- [Pull request process](#pull-request-process)
-- [Forking and cloning the project](#forking-and-cloning-the-project)
+This repository is part of MDN's learning tutorial.
```

Comment:
```suggestion
This repository is an example project to accompany MDN's HTML Basics tutorial.
```

Reply:
Thanks! I think "example project to accompany" might suggest that is an additional project. Since this is the same code used in the "Creating the content" part of the "Getting started" tutorial, perhaps we can clarify that:
```suggestion
This repository contains the code used in MDN's "[HTML: Creating the content](/en-US/docs/Learn_web_development/Getting_started/Your_first_website/Creating_the_content)" tutorial.
```